integration-cli/docker_cli_run_test.go

```diff
@@ -3948,6 +3948,7 @@ func (s *DockerSuite) TestRunAttachFailedNoLeak(c *check.C) {
 	// TODO Windows Post TP5. Fix the error message string
 	c.Assert(strings.Contains(string(out), "port is already allocated") ||
```

Comment:
I see that we don't use `gotest.tools` in this file, but this would've been cleaner with `cmp.Contains()` https://github.com/gotestyourself/gotest.tools/blob/4d8dc71d1164f5f4bb9e4fa74aa8fc30b9068aa8/assert/cmp/compare.go#L121-L129

Given that integration-cli is deprecated, probably not good to change it

Reply:
@msabansal looking at this, I noticed this block at the start of this test; is that still accurate?

```go
 	// TODO @msabansal - https://github.com/moby/moby/issues/35023. Duplicate
 	// port mappings are not errored out on RS3 builds. Temporarily disabling
 	// this test pending further investigation. Note we parse kernel.GetKernelVersion
 	// rather than system.GetOSVersion as test binaries aren't manifested, so would
 	// otherwise report build 9200.
 	if runtime.GOOS == "windows" {
 		v, err := kernel.GetKernelVersion()
 		c.Assert(err, checker.IsNil)
 		build, _ := strconv.Atoi(strings.Split(strings.SplitN(v.String(), " ", 3)[2][1:], ".")[0])
 		if build == 16299 {
 			c.Skip("Temporarily disabled on RS3 builds")
 		}
 	}
```